XamlControlsGallery/ItemPage.xaml.cs

```diff
@@ -164,13 +164,6 @@ protected override async void OnNavigatedTo(NavigationEventArgs e)
                     return;
                 }
 
-                ControlInfoDataGroup group = await ControlInfoDataSource.Instance.GetGroupFromItemAsync((string)e.Parameter);
-                var menuItem = NavigationRootPage.Current.NavigationView.MenuItems.Cast<Microsoft.UI.Xaml.Controls.NavigationViewItemBase>().FirstOrDefault(m => m.Tag?.ToString() == group.UniqueId);
-                if (menuItem != null)
```

Comment:
Where'd this stuff go?

Reply:
Earlier, when an item was selected, we selected the group associated with it. Now we can from the UI select the item or the group, so we don't have to do this anymore.

---
In reply to: [408500944](https://github.com/microsoft/Xaml-Controls-Gallery/pull/402#discussion_r408500944) [](ancestors = 408500944)